Cargo.toml

```diff
@@ -18,7 +18,7 @@ version = "0.22"
 features = ["extension-module"]
 
 [dependencies.uuid]
-version = "1"
+version = "1.11.0"
```

Comment:
This is not an upgrade in my view because "1" means "^1" which means "1.*" (https://doc.rust-lang.org/cargo/reference/resolver.html#semver-compatibility).

The resolver would use 1.11 or any latest 1.* by default, but allows other earlier versions to be used if preferred by the dependency tree of other libraries or the application. In general library dependencies should aim to be abstract and only applications lock their dependencies. If there is some new feature in 1.11+ (not in 1.0.0) that we depend on, then yes it's correct to specify 1.11, but I haven't noticed anything.